code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -406,29 +406,33 @@
 	emit_rads()
 
 /obj/machinery/atmospherics/components/unary/hypertorus/core/proc/evaporate_moderator(delta_time)
+	//Moderator total moles
+	var/moderator_internal_total_moles = moderator_internal.total_moles()
 	// Don't evaporate if the reaction is dead
 	if (!power_level)
 		return
 	// All gases in the moderator slowly burn away over time, whether used for production or not
-	if(moderator_internal.total_moles() > 0)
-		moderator_internal.remove(moderator_internal.total_moles() * (1 - (1 - 0.0005 * power_level) ** delta_time))
+	if(moderator_internal_total_moles > 0)
+		moderator_internal.remove(moderator_internal_total_moles * (1 - (1 - 0.0005 * power_level) ** delta_time))
```

Comment:
rather then doing this conversion, is there a reason this isn't just remove ratio? the exponent?

Reply:
The expression on the right would converge to 1 as delta time increases, so I have no idea why it doesn't use remove ratio.